src/main/java/gift/auth/KakaoAuth.java

```diff
@@ -86,4 +83,46 @@ public String getUserEmail(String accessToken) {
 
         return body.kakaoUserInfo().email();
     }
+
+    public void sendOrderMessage(String accessToken, OrderResponseDto orderResponseDto) {
+        String url = "https://kapi.kakao.com/v2/api/talk/memo/default/send";
+
+        String text = String.format(
+                "주문ID: %d\n옵션ID: %d\n수량: %d\n주문일시: %s\n메시지: %s",
+                orderResponseDto.id(),
+                orderResponseDto.optionId(),
+                orderResponseDto.quantity(),
+                orderResponseDto.orderDateTime(),
+                orderResponseDto.message()
+        );
+
+        String templateObject = String.format("""
+        {
+          "object_type": "text",
+          "text": "%s",
+          "link": {
+            "web_url": "https://productWeb.com", 
+            "mobile_web_url": "https://productMobileWeb.com"
+          },
+          "button_title": "확인"
+        }
+        """, text.replace("\n", "\\n").replace("\"", "\\\""));
+
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.add(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_FORM_URLENCODED_VALUE);
+        headers.add(HttpHeaders.AUTHORIZATION, "Bearer " + accessToken);
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요.  😃 

```suggestion
        headers.setBearerAuth(accessToken);
```

Reply:
위 메시지 템플릿 DTO 변경하면서 적용했습니다.